piksi_multi_cpp/include/piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler.h

```diff
@@ -12,24 +12,16 @@ const bool kLatchTopic = true;
 
 // This class handles the callback creation for the different piksi message
 // types.
-class SBPCallback {
+class SBPCallbackHandler {
  public:
+  typedef std::shared_ptr<SBPCallbackHandler> SBPCallbackHandlerPtr;
```

Comment:
Same comment as with DevicePtr ... Could also be named just Ptr, as it is never used without the Type name anyway